lib/components/print/PrintSettings.vue

```diff
@@ -4,6 +4,25 @@ v-card
     .headline Print Settings
 
   v-card-text
+    //- Selective component printing
+    v-radio-group(v-model="printSelection")
+      strong(slot="label") Print Selection
+
+      v-tooltip(bottom)
+        span(slot="activator")
+          v-radio(label="All" :value="ALL_COMPONENTS")
+        span Print all components of the game.
+
+      v-tooltip(bottom)
+        span(slot="activator")
+          v-radio(label="Select Components" :value="SELECT_COMPONENTS")
+        span Select specific components to print.
+
+    template(v-if="printSelection == SELECT_COMPONENTS")
+      v-label
+        strong Components
+      v-checkbox(v-for="component in allComponents()" :key="component.id" :label="component.title" :value="component.id" v-model="selectedComponents" hideDetails)
```

Comment:
basically it was only so that the checkboxes would show better when grouped. Without it there would be a huge space between them (probably there's more elegant ways to do this through styles and stuff).